auth/oras/client.go

```diff
@@ -0,0 +1,35 @@
+package oras
+
+import (
+	"context"
+
+	"github.com/Azure/acr-cli/version"
+	"oras.land/oras-go/v2/registry/remote"
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+// client option struct
```

Comment:
It is better to have meaningful documentation.

Reply:
removed this comment as it is quite redundant.